src/cache.cpp

```diff
@@ -150,7 +150,7 @@ namespace {
 		int min_height, max_height;
 		std::function<BitmapRef()> dummy_renderer;
 	} const spec[] = {
-		{ "Backdrop", false, 320, 320, 160, 160, backdrop_dummy_func},
+		{ "Backdrop", false, 320, 320, 160, Player::IsRPG2k() ? 160 : 240, backdrop_dummy_func},
```

Comment:
The code is wrong because the const spec[] is initialized on startup. Which means the result will be either 160 or undefined behaviour (depending on if the static initializer for the engine version runs before Cache).